### DISCUSSION TITLE
The Mach-O Transition: Darling in the Past 5 Years | Darling Development Blog

### DISCUSSION THREAD

**Discussion by giscus** [darlinghq/blog#3] [Blog Comments]
# The Mach-O Transition: Darling in the Past 5 Years | Darling Development Blog

Darling has been under development for almost five years now, which invites the questions &mdash; what has happened over the past years, are we getting anywhere and when will we get there.

https://darlinghq.github.io/blog/2017/02/the-mach-o-transition-darling-in-past-5.html

**Comment by facekapow** [comment]
This is a test comment!

**Reply by bugaevc** [reply]
Another test reply, this time through Giscus (or whaterver-it's-called) UI right under the post. Is it gonna work? Let's find out (_hits reply_)

**Reply by bugaevc** [reply]
Seems to have worked, and not let's try through GH.
